libs/shared/ofrep-core/src/lib/model/evaluation.ts

```diff
@@ -7,33 +8,36 @@ export interface EvaluationRequest {
   context?: EvaluationContext;
 }
 
-export enum EvaluationSuccessReason {
```

Comment:
This is a duplicate of some SDK constants, we don't need it and it save us tedious mapping.